### PR TITLE
Fix LXC setup

### PR DESCRIPTION
**Pull request by sauloperez**
Fixes the setup process of an LXC development environment using
playbooks/development.yml.

It relies on https://github.com/coopdevs/devenv to create the LXC
container. As it has some personal specifics I can't commit the
necessary `.devenv` file but the following can serve as an example

```
NAME="ofn"
DISTRIBUTION="ubuntu"
RELEASE="xenial"
ARCH="amd64"
HOST="ofn.local"

DEVENV_USER="openfoodnetwork"
DEVENV_GROUP="openfoodnetwork"

PROJECT_NAME="openfoodnetwork"
PROJECT_PATH="/home/pau/dev/openfoodnetwork"

PYTHON_INTERPRETER=python3
```

This is the one I used. After having the container up and running,
running the following you'll have it ready.

```
ansible-playbook playbooks/setup.yml playbooks/development.yml --limit=lxc
```

There are a number of improvements that can be made but this is a good
foundation to get up to speed.
3277bb8

### CODE REVIEW

File: playbooks/development.yml [hunk 1]
```diff
@@ -1,4 +1,12 @@
 ---
+# Note that by relying on this playbook you will end up with a development
+# environment that uses some of roles we use for live instances. You won't have
+# Nginx nor unicorn so will have to manually start the rails running `bundle
+# exec rails server`.
+#
+# Also, there are a number of things you will need to do manually such as
+# installing bundler and creating the development database with `bundle exec
```
Comment (Matt-Yorkley) [inline playbooks/development.yml]: P.S. If you want to provision a dev setup without requiring _any_ manual steps from the user, take a look here: https://github.com/openfoodfoundation/ofn-devbox/blob/master/setup/provision.yml

**Comment by enricostano** [comment]
@Matt-Yorkley you're more than welcome to open issues on https://github.com/coopdevs/devenv , we're open to discuss implementation and improve things over there :muscle:

**Comment by Matt-Yorkley** [comment]
LXD is the answer for managing CRUD and networking configuration of linux containers in a clean way. I started looking at it yesterday, but I had to stop because the current version isn't working on Debian Buster :see_no_evil:

**Comment by sauloperez** [comment]
> Debian Buster 

Don't do this to you guys :trollface:

**Comment by Matt-Yorkley** [comment]
@sauloperez could you add a quick Wiki entry for basic setup and usage of the LXC scripts?

**Comment by enricostano** [comment]
> As it has some personal specifics I can't commit the necessary .devenv file

@sauloperez why? would be nice (and easier) to have a generic OFN development `.devenv`, right?

**Comment by sauloperez** [comment]
My concern was to have my specific project path defined there. We can't expect everyone to share the same folder structure, do we? Perhaps something to customize through env vars?

**Pull request merged**